src/locales/en/common.json

```diff
@@ -577,6 +582,11 @@
     "selectAll": "Select all"
   },
   "account": {
+    "tokens": {
+      "contractAddress": "Contract address",
+      "viewInExplorer": "View in explorer",
+      "howTo": "To add tokens accounts, you need to <0>receive funds</0> through your <1>Ethereum address</1>."
```

Comment:
@dasilvarosa could you check these? i'm not sure about the wording. `through your Ethereum address`